<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>connectionSettings</class>
 <widget class="QWidget" name="connectionSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection Settings</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>icons/setting-icon.png</normalon>
   </iconset>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="verticalGroupBox">
     <property name="title">
      <string>Connection</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="connectionTypeBox">
        <item>
         <property name="text">
          <string>Modbus TCP/IP</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Modbus Serial</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="verticalGroupBox">
     <property name="title">
      <string>TCP/IP Connection Settings</string>
     </property>
     <layout class="QVBoxLayout" name="tcpipSettingsVBoxLayout">
      <item>
       <widget class="QLabel" name="ipAddressLabel">
        <property name="text">
         <string>IP Address</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ipAddressEdit">
        <property name="text">
         <string>192.168.0.131</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <widget class="QLineEdit" name="serverPortEdit">
          <property name="text">
           <string>502</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="serverPortLabel">
          <property name="text">
           <string>Server Port</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="connectionTimeoutEdit">
          <property name="text">
           <string>3000</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="connectionTimeoutLabel">
          <property name="text">
           <string>Connection Timeout [ms]</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>96</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
